class/class-feature.php

```diff
@@ -0,0 +1,261 @@
+							<div class="">
+								<?php echo esc_html( $user->user_nicename ); ?>
+							</div>
+
+							<!--  feature uploaded  time -->
+							<div class="">
+								<?php $time_ago = self::get_time( $feature->time ); ?>
+
+								<p><?php echo esc_html( $time_ago ); ?></p>
+							</div>
+
+							<div class=" larger-item project-box">
+								<span class="text-box">
+									<?php echo esc_html( ucwords( str_replace( '-', ' ', $feature->project_status ) ) ); ?>
+								</span>
+							</div>
+							<div class=" project-box">
+								<span class="text-box">
+									<?php echo esc_html( ucwords( str_replace( '-', ' ', $feature->project_priority ) ) ); ?>
+								</span>
+							</div>
+							
+						</div>
+					</div>
+				</div>
+			<?php
+		}
+	}
+
+	/**
+	 * Get time.
+	 *
+	 * @param string $datetime time.
+	 * @return string
+	 */
+	public static function get_time( $datetime ) {
+		// Get the site's timezone from WordPress settings.
+		$timezone  = get_option( 'timezone_string' );
+		$date_time = new DateTime( $datetime );
+
+		$date = $date_time->format( 'd M Y' );
+		// echo $datetime;
+		// Create a DateTime object with the provided datetime and set the timezone.
+		$datetime_obj = new DateTime( $datetime, new DateTimeZone( $timezone ) );
+		// Calculate the time difference between the provided datetime and now.
+		$now      = new DateTime( 'now', new DateTimeZone( $timezone ) );
+		$interval = $datetime_obj->diff( $now );
+
+		// Get the human-readable time difference.
+		$human_time_diff = human_time_diff( $datetime_obj->getTimestamp(), $now->getTimestamp() );
+
+		// Format the result as "X time ago".
+		if ( $interval->y > 0 ) {
+			return $date;
+		} elseif ( $interval->m > 0 ) {
+			return $date;
+		} elseif ( $interval->d > 0 ) {
+
+			return $date;
+		} elseif ( $interval->h > 0 ) {
+			return $human_time_diff . ' ago';
+		} elseif ( $interval->i > 0 ) {
+			return $human_time_diff . ' ago';
+		} else {
+			return 'just now';
+		}
+	}
+
+	/**
+	 * Feature vote.
+	 *
+	 * @param string $feature_id feature id.
+	 */
+	public static function feature_vote( $feature_id ) {
+		?>
+		<div class="vote-count ">
+			<span id="<?php echo esc_html( 'like' . $feature_id ); ?>">
+				<?php
+					$count = Database::get_all_votes( $feature_id );
+					echo esc_html( $count );
+				?>
+			</span>
+		</div>
+
+		<?php
+		self::vote_btn( $feature_id );
+	}
+
+	/**
+	 * Its add vote button.
+	 *
+	 * @param int $feature_id is an feature id of product.
+	 */
+	public static function vote_btn( $feature_id ) {
+		?>
+		<div class="vote-grp">
+			<?php if ( Database::already_liked_user( $feature_id ) ) : ?>
+				<div class="btn-grp vote-btn-div" id="<?php echo esc_html( 'bg' . $feature_id ); ?>" style="background-color:white; color:black;" >
+					<button type="button" class="vote-btn btn-vote"  id = "<?php echo esc_html( $feature_id ); ?>" onclick="vote(event,<?php echo esc_html( $feature_id ); ?>)">
+						Vote
```

Comment:
Static texts should be echoed through a translation function to support WordPress's default translation system.
Please refer following articles for more
https://developer.wordpress.org/reference/functions/esc_html__/
https://codex.wordpress.org/I18n_for_WordPress_Developers